nebula-unity/Assets/Scripts/CharacterSuper.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CharacterSuper : MonoBehaviour
+{
+    // Start is called before the first frame update
+    public float hp;
+    public float maxhp;
+    public float moveSpeed;
+
+    void Start()
+    {
+        
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+
+    void TakeDamage(float damage){
```

Comment:
We should think about sanitizing inputs on some of these functions.  What happens if I call TakeDamage() with a negative number?  Is healing taken care of here?  If so maybe we should rename this to ModifyHealth() or something.